packages/core-mobile/e2e/tests/plusIcon/parameterizedSend.e2e.ts

```diff
@@ -0,0 +1,29 @@
+import AccountManagePage from '../../pages/accountManage.page'
+import sendPage from '../../pages/send.page'
+import { warmup } from '../../helpers/warmup'
+import { Tokens } from '../../helpers/tokens'
+import popUpModalPage from '../../pages/popUpModal.page'
+import actions from '../../helpers/actions'
+import loginRecoverWallet from '../../helpers/loginRecoverWallet'
+
+describe('Send parameterized tokens', () => {
+  beforeAll(async () => {
+    await warmup()
+    await AccountManagePage.createSecondAccount()
+  })
+
+  Tokens.forEach(async ({ symbol }) => {
+    test(`should send ${symbol}`, async () => {
+      if (device.getPlatform() === 'android') {
+        await device.launchApp({ newInstance: true })
+      } else {
+        await device.reloadReactNative()
```

Comment:
this won't work in CI since we run the ios tests in parallel.  it reloads react native for all of the running devices.